### PR TITLE
Some fixes for integration tests with tester

### LINKED ISSUE #1865: test_participant_selection fails when using the tester backend
As produced by `tests/integration/long_running/flaky/test_token_networks.py`

```
Traceback (most recent call last):
  File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
  File "/home/xoza/src/raiden/raiden/tasks.py", line 56, in _run
    self.poll_for_new_block()
  File "/home/xoza/src/raiden/raiden/tasks.py", line 95, in poll_for_new_block
    result = callback(current_block, chain_id)
  File "/home/xoza/src/raiden/raiden/raiden_service.py", line 392, in _callback_new_block
    self.handle_state_change(state_change, current_block_number)
  File "/home/xoza/src/raiden/raiden/raiden_service.py", line 340, in handle_state_change
    on_raiden_event(self, event)
  File "/home/xoza/src/raiden/raiden/raiden_event_handler.py", line 352, in on_raiden_event
    handle_contract_send_channelsettle(raiden, event)
  File "/home/xoza/src/raiden/raiden/raiden_event_handler.py", line 311, in handle_contract_send_channelsettle
    second_locksroot,
  File "/home/xoza/src/raiden/raiden/network/proxies/payment_channel.py", line 169, in settle
    partner_locksroot=partner_locksroot,
  File "/home/xoza/src/raiden/raiden/network/proxies/token_network.py", line 786, in settle
    raise TransactionThrew('Settle', receipt_or_none)
raiden.exceptions.TransactionThrew: Settle transaction threw. Receipt=AttributeDict({'transactionHash': HexBytes('0x07763bfc6e8538356eaa21f581898b3dd01b05df553a56ef8f5cd363a6aacb73'), 'transactionIndex': 0, 'blockNumber': 125, 'blockHash': HexBytes('0xd4a68dd33499c58119708b1d07496dfe5845ff0cc2cf5d5345083ed2d8fee788'), 'cumulativeGasUsed': 30228, 'gasUsed': 30228, 'contractAddress': None, 'logs': [], 'status': 0})
2018-07-18T08:24:08Z <AlarmTask "Greenlet-0" at 0x7f8ecb13f248: _run> failed with TransactionThrew
```

Channel is closed at block 117 and settle is called at block 125. Settle timeout should be 6 blocks.

### PR DESCRIPTION
**Pull request by andrevmatos**
Geth and parity seems to not have problems with `[<topic>, [<topic>, <topic>]]` topics list, but it's not in the spec and tester crashes with it. Simplest fix is to, when using list of lists, be consistent with that.
Together with #2015, fixes #1865

### CODE REVIEW

File: raiden/log_config.py [hunk 1]
```diff
@@ -252,4 +252,5 @@ def configure_logging(
 
     # set logging level of the root logger to DEBUG, to be able to intercept
     # all messages, which are then be filtered by the `RaidenFilter`
-    structlog.get_logger('').setLevel('DEBUG')
+    structlog.get_logger('').setLevel(logger_level_config.get('', DEFAULT_LOG_LEVEL))
+    structlog.get_logger('raiden').setLevel('DEBUG')
```
Comment (pcppcp) [inline raiden/log_config.py]: default is ok for Raiden too - the debug can be found in the log file

File: raiden/network/rpc/client.py [hunk 2]
```diff
@@ -283,7 +283,7 @@ def get_transaction_receipt(self, tx_hash: bytes):
         return self.web3.eth.getTransactionReceipt(encode_hex(tx_hash))
 
     def get_transaction_from(self, tx_hash: bytes) -> Optional[Address]:
-        receipt = self.web3.eth.getTransactionReceipt(encode_hex(tx_hash))
+        receipt = self.web3.eth.getTransaction(encode_hex(tx_hash))
```
Comment (pcppcp) [inline raiden/network/rpc/client.py]: why this change?

Comment (andrevmatos) [reply raiden/network/rpc/client.py]: tester `getTransactionReceipt` doesn't have `from` key. transaction does.

File: raiden/log_config.py (hunk 1 above)
Comment (andrevmatos) [reply raiden/log_config.py]: That's why this change: we need to set `DEBUG` at the logger level, to be able to catch everything on logger (and later **filter** separately for the debug file and standard **handlers**). So, before, everything was always `DEBUG`, so the handlers filters were the only responsible for doing this filtering. But, as pytest captures every logging call, it spammed travis logs with debug logs of all modules. This change still keeps the filters working on the handlers level, but also sets **some** level to the root logger (taking into account if user wants root to be debug too, with `--log-config`), to avoid travis getting overwhelmed. IOW: this sets the logger levels to `logger_level_config` _less-restrict_ levels, while still catching what needed for the main filter.

**Comment by andrevmatos** [comment]
#2738 removed tester tests
